src/wp-content/themes/twentynineteen/functions.php

```diff
@@ -245,6 +245,13 @@ function twentynineteen_content_width() {
 	// This variable is intended to be overruled from themes.
 	// Open WPCS issue: {@link https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1043}.
 	// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound
```

Comment:
I think these three single-line comments can be removed from both themes. The WPCS issue is not open anymore, and https://github.com/WordPress/WordPress-Coding-Standards/pull/1773 indicates that the test makes an exception for `$content_width`.